book-my-show/backend/java/bms-monolith/src/main/java/org/lbcc/bms/bms_monolith/entity/EventShow.java

```diff
@@ -34,11 +32,14 @@ public class EventShow extends BaseAuditingEntity {
     @Enumerated(EnumType.STRING)
     private List<Genre> genres;
 
+    @NotNull
     private Instant startDate;
+
+    @NotNull
```

Comment:
@NotNull will enforce a not null check from the application side, but it will not create a NOT NULL constraint on the database end. Bothe has to be added.

What is missing here is the Request, Response, and DTO layers.
@NotNull and similar spring validation should be in Request layer

Entity should have @Column(nullable = false)